backend/dataall/base/utils/api_handler_utils.py

```diff
@@ -0,0 +1,47 @@
+import json
+
+from dataall.base.services.service_provider_factory import ServiceProviderFactory
+
+
+def get_cognito_groups(claims):
+    if not claims:
+        raise ValueError(
+            'Received empty claims. ' 'Please verify authorizer configuration',
+            claims,
+        )
+    groups = list()
+    saml_groups = claims.get('custom:saml.groups', '')
+    if len(saml_groups):
+        groups: list = saml_groups.replace('[', '').replace(']', '').replace(', ', ',').split(',')
```

Comment:
Could also possibly use a translation table here instead of the successive replace statements?

<img width="457" alt="Screenshot 2024-05-03 at 4 17 30 PM" src="https://github.com/data-dot-all/dataall/assets/97474536/9ec165db-e519-4ef9-a2bd-79ffad2d9d5a">



`
translation_table = str.maketrans({'[': None, ']': None, ', ': ','})
groups = []
saml_groups = claims.get('custom:saml.groups', '')

if len(saml_groups) > 0:
    groups = saml_groups.translate(translation_table).split(',')
`

Reply:
Yes. I can do this one! More cleaner